### ISSUE TITLE
Support `onError` hook during the `init` phase

### ISSUE THREAD

**Issue by sobolevn** [nuxt-community/auth-module#220]
### What problem does this feature solve?

Currently the suggested way to bind your own error hooks is by defining a custom plugin:

```js
// plugins/auth.js
export default function({ $auth }) {
  $auth.onError((error, name, endpoint) => {
    console.error(name, error)
  })
}
```

And then extending the auth config like so:

```js
{
  modules: {
    '@nuxtjs/auth'
  },
  auth: {
     plugins: [ '~/plugins/auth.js' ]
  }
}
```

But, the problem is that when some error happens during `$auth.init()`. 
In this case `_errorListeners` is not yet populated, so there's no way to intercept this error.

In my case this is token expiration error from one of the `oauth` providers.

How about providing any mechanism to handle errors during `init()`?

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c177">#c177</a>)</em></sub></div>

**Comment by gijswijs** [comment]
I have this exact same issue. I want to handle an error during init() gracefully, but there's no proper way to load error listeners before the Init.

**Comment by gijswijs** [comment]
I though it would be best to just go ahead and create a PR myself.

**Comment by pi0** [comment]
This should be fixed in [v4.6.0](https://github.com/nuxt-community/auth-module/releases/tag/v4.6.0) by #234. Please ping here if the issue still exists.
